app/javascript/admin/controllers/modal_controller.js

```diff
@@ -0,0 +1,10 @@
+import { Controller } from 'stimulus';
+
+export default class ModalController extends Controller {
+  toggleModal() {
+    let modal = document.getElementById(this.data.get('target-id'));
```

Comment:
refactor (non-blocking): Just wondering why we're not using Stimulus' [targets](https://stimulusjs.org/reference/targets) via the `data-target` attribute.

Also, is `target-id` being used instead of `modal-target-id` because there is some naming convention wizardry going on?

Reply:
Yeah, I had some trouble using targets for this the first time through, but looking at it with fresh eyes, that is the right call. I just needed to add some kind of container element so that the same target could be shared by both buttons.

Reply:
To the second question, yes! :+1: